Flexiana/framework
# State events

## Title: 

Event resourcing example. 

### Description
The actual resource is the aggregate of the events. You can create a resource with `PUT` modify it with `POST` and mark as deleted with `DELETE` methods. Every events can be undone except creation, or the last change was from another session. Undo events can redoed in (with) the same session(-id). It possible to remove fields one by one (except resource id, and resource type), and completely clean up a resource.

### Tester info
Start the application with
 - Start postgres with `docker-compose up -d`
 - `lein release` and `lein run` commands
 - open [the app](http://localhost:3000/re-frame)
 - Add new person, select it from the list above, try to apply different changes on it
 - Open another session in private browser, or another browser, to see if SSE works
 - check for cookies in the developer tools, it shoud have a session-id key with UUID value
